src/lib/components/Authenticate/index.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import Login from './Login.svelte';
+  import SignUp from './SignUp.svelte';
+
+  let login = false;
+  function handleChangeLogin() {
+    if (login == false) {
+      login = true;
+    } else if (login == true) {
+      login = false;
+    }
+  }
```

Comment:
We only have 2 forms and 2 possible values. A boolean fits very well for this implementation. I suggest relying on a use case and discard the other for simplicity.

```suggestion
  let isLogin = true;

  function handleToggleForm() {
    isLogin = !isLogin;
  }
```